javascript-bank/account.js

```diff
@@ -0,0 +1,44 @@
+/* exported Account */
+
+function Account(number, holder) {
+  this.number = number;
+  this.holder = holder;
+  this.transactions = [];
+}
+
+Account.prototype.deposit = function (amount) {
+  var trueOrFalse = true;
+  if (amount > 0 && typeof amount === 'number' && Number.isInteger(amount)) {
```

Comment:
You don't need to check both `typeof` and `isInteger`. You can just use `isInteger` because if `amount` isn't a number then `isInteger` will return `false`:

```JAVASCRIPT
if (Number.isInteger(amount) && amount > 0)
```

Also I would check if it is a integer before checking if amount is greater than 0.